lib/resilient/circuit_breaker/registry.rb

```diff
@@ -0,0 +1,33 @@
+module Resilient
+  class CircuitBreaker
+    class Registry
```

Comment:
Feels weird to me not to use a `Hash` to start, since `Hash#clear` and `Hash#fetch` cover the interface. I assume you're breaking out a wrapper class to make the interface explicit.